### PR TITLE
Lottie 1.0 release news page

### PR DESCRIPTION
**Pull request by b-wils**

### CODE REVIEW

File: content/news/announcing-lottie-specification-1.0/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: Lottie Animation Community announces Lottie v1.0 specification
+date: "2024-09-17T12:01:01.284Z"
+description: " The Lottie Animation Community (LAC), a Joint Development Foundation project, is pleased to announce the release of the v1.0 Lottie JSON specification."
+featuredImage: lottie-1_0-release.png
+---
+
+### Lottie Animation Community announces Lottie v1.0 specification
+
+The Lottie Animation Community (LAC), a [Joint Development Foundation project](https://jointdevelopment.org/), is pleased to announce the release of the v1.0 Lottie JSON specification.
+This marks a significant milestone in our efforts to establish the [Lottie File Format](https://lottie.github.io/) as an open standard for efficient, scalable, and cross-platform animated vector graphics.
```
Comment (mbasaglia) [inline content/news/announcing-lottie-specification-1.0/index.md:11]: Should "Lottie File Format" link to the specs rather than the main website?

Comment (b-wils) [reply content/news/announcing-lottie-specification-1.0/index.md:11]: Yeah I think that makes sense. Should we wait for https://github.com/lottie/lottie-spec/pull/96 to get merged so we can use the static 1.0 URL?

Comment (mbasaglia) [reply content/news/announcing-lottie-specification-1.0/index.md:11]: There's a few different options:

* https://lottie.github.io/lottie-spec/ directs to the default version (which I set to be `latest`)
* https://lottie.github.io/lottie-spec/latest/ is an alias, currently pointing to `dev` but will be set to `1.0` once released
* https://lottie.github.io/lottie-spec/1.0/ will be the URL to the 1.0-specific version

**Review by kudanai — APPROVED** [review]

File: content/news/announcing-lottie-specification-1.0/index.md (hunk 1 above)
Comment (b-wils) [reply content/news/announcing-lottie-specification-1.0/index.md]: I used https://lottie.github.io/lottie-spec/1.0/ so this page will always point there, even after subsequent versions are released.

**Review by mbasaglia — APPROVED** [review]

**Review by Aidosmf — APPROVED** [review]

**Pull request merged**